shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/api/GovernanceRepositoryAPI.java

```diff
@@ -41,9 +40,11 @@ public interface GovernanceRepositoryAPI {
     /**
      * Persist job progress.
      *
-     * @param jobContext job context
+     * @param jobId job id
+     * @param shardingItem sharding item
+     * @param progressValue progress value
      */
-    void persistJobProgress(PipelineJobContext jobContext);
+    void persistJobProgress(String jobId, int shardingItem, String progressValue);
     
     /**
      * Get job progress.
```

Comment:
It's better to refactor `JobProgress getJobProgress(String jobId, int shardingItem);` too